benchmarks/base.py

```diff
@@ -55,8 +45,29 @@ def random_data_dir(num_files, file_size):
     if not os.path.exists(dir_path):
         os.makedirs(dir_path, exist_ok=True)
         filenames = [
-            os.path.join(dir_path, "file_{}".format(str(i))) for i in range(num_files)
+            os.path.join(dir_path, "file_{}".format(str(i)))
+            for i in range(num_files)
         ]
-        with ThreadPoolExecutor(max_workers=max(cpu_count() / 2, 2)) as executor:
+        with ThreadPoolExecutor(
+            max_workers=max(cpu_count() / 2, 2)
+        ) as executor:
             executor.map(random_file, filenames, len(filenames) * [file_size])
     return dir_path
+
+
+class BaseBench:
+    def setup(self):
+        self.cwd = os.getcwd()
+        self.test_directory = TemporaryDirectory(prefix="DVCBenchmark")
+        os.chdir(self.test_directory.name)
+
+    def teardown(self):
+        os.chdir(self.cwd)
+        self.test_directory.cleanup()
+
+
+class StartupBench(BaseBench):
```

Comment:
Why is it in `base` though? Would it make sense to create a separate file for startup benchmark?